docs/source/user-guide/python.md

```diff
@@ -103,14 +119,15 @@ The `explain` method can be used to show the logical and physical query plans fo
 The following example demonstrates creating arrays with PyArrow and then creating a Ballista DataFrame.
 
 ```python
-import ballista
+from ballista import Ballista, BallistaBuilder
 import pyarrow
 
 # an alias
+# TODO implement Functions
 f = ballista.functions
 
 # create a context
-ctx = ballista.BallistaContext("localhost", 50050)
+ctx = Ballista.standalone()
```

Comment:
this line is not correct

Reply:
fixed, changed to `BallistaBuilder().standalone()`